test/smoke.spec.js

```diff
@@ -0,0 +1,53 @@
+var expect = require('chai').expect;
+var sinon = require('sinon');
+var sh = require('shelljs');
+var path = require('path');
+var basePath = path.resolve(__dirname, '..');
+var helper = require('./generateEsLintResult');
+var fs = require('fs-extra');
+var pathToTestJson = path.resolve(__dirname, 'result.json');
+var pathToIndex = path.resolve(__dirname, '..', 'index.js');
+var input = [];
```

Comment:
You need to clear out the input array otherwise each test is going to keep adding an a growing array (in the `before` callback). You probably want to just have `var input;` here and then `input = [];` in the `before` callback.